lib/addons/OPA/index.ts

```diff
@@ -0,0 +1,14 @@
+import { ClusterAddOn, ClusterInfo } from "../../stacks/cluster-types";
+
+export class OpaAddOn implements OpaAddOn {
```

Comment:
If it is gatekeeper that we install, a better name would be `OpaGatekeeperAddOn` (based on https://github.com/open-policy-agent/gatekeeper#how-is-gatekeeper-different-from-opa)

Reply:
Addressed

Reply:
Is it? OPA directory is still upper case. OpaAddon no renamed?

Reply:
changed name of directory to opa-gatekeeper and the addon is now named OpaGatekeeperAddOn - let me know if you'd like me to rename